docs/_common/cluster-setup-for-visual-tutorials.md

```diff
@@ -0,0 +1,95 @@
+Below are instructions for setting up a Kubernetes cluster with network policies.
+If you don't have a cluster already, we recommend starting out with a Minikube cluster.
+
+<Tabs groupId="cni">
+<TabItem value="minikube" label="Minikube">
+
+If you don't have the Minikube CLI, first [install it](https://minikube.sigs.k8s.io/docs/start/). 
+
+Then start your Minikube cluster with Calico, in order to enforce network policies.
+
+```bash
+minikube start --cpus=4 --memory 8192 --disk-size 32g --cni=calico
+```
+The minikube system requirements are for the demo application we deploy in the visual tutorials.
```

Comment:
Remember to use correct capitalization here, capital M for Minikube - it's a name, same as the line before the setup instructions.

Also, I think it might not be clear what you mean with "system requirements", since they are requirements of the visual tutorial app, not of Minikube. So what do you think about something like: "The increased CPU, memory and disk resource allocations are required to be able to deploy the ecommerce app used in the visual tutorials successfully."